builder/azure/arm/testdata/child_from_specialized_parent.pkr.hcl

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+
+variable "ssh_password" {
+  type = string
+  default = "${env("ARM_SSH_PASS")}"
+  sensitive = true
+}
+
+variable "subscription" {
+  default = "${env("ARM_SUBSCRIPTION_ID")}"
+  type = string
+  sensitive = true
+}
+
+variable "ssh_private_key_location" {
+  default = "${env("ARM_SSH_PRIVATE_KEY_LOCATION")}"
+  type = string
+}
+
+source "azure-arm" "linux-sig" {
+  use_azure_cli_auth = true
+  location           = "South Central US"
+  vm_size            = "Standard_D4ps_v5"
+  ssh_username       = "packer"
+  ssh_password       = var.ssh_password
+  ssh_private_key_file = var.ssh_private_key_location
+  communicator       = "ssh"
+  shared_image_gallery{
+    subscription   = var.subscription
+    image_name     = "arm-linux-specialized-sig"
+    gallery_name   = "acctestgallery"
+    image_version  = "1.0.0"
+    resource_group = "packer-acceptance-test"
+  }
+  shared_image_gallery_destination {
+    image_name     = "arm-linux-specialized-sig"
+    gallery_name   = "acctestgallery"
+    image_version  = "1.0.1"
+    resource_group = "packer-acceptance-test"
+    specialized    = true
```

Comment:
Seeing as the parent image is specialized does the child honor the specialized property. In other words, if parent is specialized the child will be specialized? Or does the user have to explicitly set specialized to true?

Reply:
The parent being specialized does not impact the child being specialized it just impacts the Virtual Machine you create.

You can create a specialized child or a generalized child from a specialized parent.

Reply:
Got it thanks for the explanation